sdk/src/main/java/com/mapbox/maps/renderer/MapboxTextureViewRenderer.kt

```diff
@@ -7,9 +7,17 @@ import androidx.annotation.VisibleForTesting
 
 internal class MapboxTextureViewRenderer : MapboxRenderer, TextureView.SurfaceTextureListener {
 
+  override val widgetRenderer: MapboxWidgetRenderer
+
   constructor(textureView: TextureView, antialiasingSampleCount: Int) {
+    val widgetRenderer = MapboxWidgetRenderer(
+      translucentSurface = false,
```

Comment:
any sense in this arg? it's always false, although for texture renderer we have it as `true`